### PR TITLE
Docs versioning

### PR DESCRIPTION
**Pull request by rainum**
#12

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ gulp.task('styles', function () {
   return gulp.src('app/_assets/stylesheets/index.less')
     .pipe($.plumber())
     .pipe($.less())
-    .pipe($.uncss({ html: glob.sync('dist/**/*.html') }))
+    //.pipe($.uncss({ html: glob.sync('dist/**/*.html') }))
```
Comment (thibaultCha) [inline gulpfile.js]: This is very important, why removing it?

Comment (orliesaurus) [inline gulpfile.js]: https://github.com/Mashape/getkong.org/commit/14e2f69d643434f11e57cd3a8456ad11b612f354

Comment (thibaultCha) [inline gulpfile.js]:
```
options = {
  ignore: ['#added_at_runtime', /test\-[0-9]+/],
};
```

**Comment by thefosk** [comment]
@rainum can we close this?

**Comment by Nijikokun** [comment]
This removes the `latest` alias that we use to refer to the latest documentation.

Pinging @montanaflynn and @rainum you guys can discuss this further as @rainum removed the functionality.

**Comment by thibaultCha** [comment]
It shouldn't, according to the code it's added by the Ruby plugin.

**Comment by Nijikokun** [comment]
> It shouldn't, according to the code it's added by the Ruby plugin.

Ah I missed this section. Looks good otherwise.

**Comment by thibaultCha** [comment]
Well except for the step back removing uncss.

**Comment by Nijikokun** [comment]
> Well except for the step back removing uncss.

It's a small thing, just need @rainum to add ignore option for these classes.

**Comment by thibaultCha** [comment]
It actually doesn't work. I see no dropdown even with 2 versions and the `latest` redirection is gone, there is only the one on `/docs/`

**Comment by Nijikokun** [comment]
> It actually doesn't work.

I was fearful of this, when I first implemented the `alias` functionality I was going to do it automatically in this file but was afraid it would not work. *sigh*

**Comment by montanaflynn** [comment]
Wait, what's going on? Don't deploy this if it breaks all the `/latest/` links!

**Comment by thibaultCha** [comment]
It doesn't because they were removed from 0.2.0-2 but they are still in 0.2.1.

**Pull request merged**